### PR TITLE
Completed PreCourse1 - Please review

### PR DESCRIPTION
**Pull request by Ameya145**

### CODE REVIEW

File: LinkedList.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // a Singly Linked List 
 public class LinkedList { 
   
-    Node head; // head of list 
+    static Node head; // head of list
```
Comment (super30admin) [inline LinkedList.java]: Could you please state the reason for head being static here?

**Comment by super30admin** [comment]
Well Done!! You have completed the assigned tasks correctly.
Your solutions suggest that you seem to be good at Stacks and Linked List.
Suggestion:
-Try to Comment your approach to the solution along with the code.

File: LinkedList.java (hunk 1 above)
Comment (Ameya145) [reply LinkedList.java]: Oh! There is no reason as such. I was getting warnings from Eclipse regarding this, hence I changed. 
Making it static did not affect my solution as such.

Comment (super30admin) [reply LinkedList.java]: Would it work if you try to make more than one Linked Lists using this class ?

**Pull request closed without merging**